quizapp/src/main/resources/application-dev.yml

```diff
@@ -1 +1,14 @@
-allowedOrigin: ""
\ No newline at end of file
+allowedOrigin: "http://localhost:4200/"
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    password: ${DB_PASSWORD:admin}
```

Comment:
pierwszy raz się spotykam z taka semnatyką. co oznacza? `${DB_PASSWORD:admin}`  `${DB_USERNAME:postgres}` `${DB_URL:jdbc:postgresql://127.0.0.1:5432/}`

Reply:
to jest zmienna środowiskowa czyli jesli przy uruchomieniu podasz inne hasło to wstawi to co podałeś a jesli nie podasz wybierze to po dwukropku.

Reply:
aaa no tak :p